planner/core/plan.go

```diff
@@ -341,6 +341,10 @@ type PhysicalPlan interface {
 	// Stats returns the StatsInfo of the plan.
 	Stats() *property.StatsInfo
 
+	Cost() float64
+
+	SetCost(cost float64)
```

Comment:
Please add some comments otherwise the linter would be unhappy.

Reply:
Ok